recipes/slamdunk/meta.yaml

```diff
@@ -14,7 +14,7 @@ build:
     - slamdunk=slamdunk.slamdunk:run
     - alleyoop=slamdunk.alleyoop:run
     - splash=slamdunk.splash:run
-  number: 0
+  number: 1
   skip: True # [py3k or osx]
```

Comment:
Any reason for not building for py3k or osx? It seems to build fine locally on my mac.

Reply:
For earlier versions, the osx and py3k builds never worked, so I decided to exclude them. Have never tried since then, but if you tell me they work for you, I can definitely include them again.